docs/extend/modular-resources/examples/csi.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Add a CSI Camera as a Modular Resource"
+linkTitle: "CSI Camera"
+weight: 40
+type: "docs"
+description: "How to add a CSI Camera as a modular resource of your robot."
+tags: ["board", "csi", "jetson", "serial", "module", "modular resources", "Python", "python SDK", "nvidia", "jetson orin", "jetson orin nano", "nano", "camera"]
+# SMEs: Sean
+---
+
+
+Viam provides a modular resource [extending](/extend/modular-resources/) the [camera API](/components/camera/#api) as a new `viam:camera:csi` model of [camera](/components/camera/).
+
+This module includes a simple wrapper around `GStreamer` and a control interface for the **control** tab of the [Viam app](https://app.viam.com) so you can utilize the hardware accelerated GST plugins and use the embedded CSI cameras on your `jetson` boards with Viam.
+
+The module is open-sourced and available on [GitHub](https://github.com/seanavery/viam-csi).
+
+[Install requirements](#requirements) and [configure](#configuration) the module to add an `viam:camera:csi` [camera](/components/camera/) {{< glossary_tooltip term_id="resource" text="resource" >}} to your robot.
+
+## Requirements
+
+On your robot's computer, download [the `viam:camera:csi` appimage](https://github.com/viamrobotics/odrive) and make it executable:
+
+``` {class="command-line" data-prompt="$"}
+sudo wget https://github.com/seanavery/viam-csi/releases/download/v0.0.2/viam-csi-0.0.2-aarch64.AppImage -O /usr/local/bin/viam-csi
+sudo chmod 755 /usr/local/bin/viam-csi
+```
+
+## Configuration
+
+### Module
+
+{{< tabs name="Add the csi module">}}
+{{% tab name="JSON Template" %}}
+
+Go to your robot's page on the [Viam app](https://app.viam.com/).
+Navigate to the **Config** tab on your robot's page and select **Raw JSON** mode.
+Copy and paste the following raw JSON to add a `csi` [camera](/components/camera/) component with the name `my_test_csi_cam`:
+
+```json {class="line-numbers linkable-line-numbers"}
```

Comment:
it might be good to separate this and ask them to first add the module to the modules array and then the component to the components array - just to ensure people don't copy this and add it to the raw json, resulting in two components keys or osmething like that

Reply:
@seanavery built this for users in the repo and I just copied/pasted-- assume it was for ease of configuration and not having to add the module separately, but splitting out would make a lot more sense if I also added a tab for config builder. @seanavery is it ok to split out and do that?

Reply:
that's only easy if there's no other modules/components present though ;)

Reply:
@sguequierre Yes, it should be ok to split up the config for clarity in the documentation. I think it still makes sense to keep example config as single json file in the module repo.

Reply:
@sguequierre let's make this tabbed then, with a template and an example

Reply:
@npentrel Ok, sounds good! Thanks guys!

Reply:
@npentrel Ok, sounds good! Thanks guys! Will comment when done

Reply:
@npentrel Tabbed with template and example

Reply:
@npentrel Done I beleive